python/tank/platform/engine.py

```diff
@@ -910,7 +910,7 @@ def register_command(self, name, callback, properties=None):
 
         - ``group_default`` - Boolean value indicating whether this command should represent the group
           as a whole. Setting this value to True indicates that this is the command to run and display
-          in applications that show a single button for each command group.
+          in applications that show a single representation for any command group.
```

Comment:
Here's what I came up with: 

```Cases where multiple or no group_default values are specified for a command group should be handled accordingly by the bundle parsing the registered commands.```

Should Desktop's implementation be mentioned as an example?